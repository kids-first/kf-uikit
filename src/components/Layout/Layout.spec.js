/* eslint-disable */
import React from 'react';
import { Box, Flex } from '.';

test('Box renders with correct styles', () => {
  const actual = create(<Box />);
  expect(actual).toMatchSnapshot();
});

test('Flex renders with correct styles', () => {
  const actual = create(<Flex />);
  expect(actual).toMatchSnapshot();
});
