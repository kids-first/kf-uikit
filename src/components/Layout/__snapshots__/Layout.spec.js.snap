// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Box renders with correct styles 1`] = `
.Box, Box {
  border: 1px solid black;
}

<div
  className="Box, Box"
/>
`;

exports[`Flex renders with correct styles 1`] = `
.Box__flex, Box__flex {
  border: 1px solid black;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

<div
  className="Box__flex, Box__flex"
/>
`;
